GoogleForCreators/web-stories-wp
# Change the text wrapping for text display.

## Summary

Changes the Text wrapping for a text element.
Per [Figma](https://www.figma.com/file/Cd9iacVknHK62RO4pRwb68/Stories-Editor?node-id=8011%3A14593), when Text's width is resized smaller, the words shouldn't overflow.

We had the text just overflowing but only in the display mode which caused differences between the edit mode and the display mode.



## Relevant Technical Choices

- Rounds width and height used for display mode since otherwise there are differences with the measurer which can cause the height not being accurately calculated.


## To-do



## User-facing changes
- When text element's width is resized, the words will be wrapped when they don't fit in length.


## Testing Instructions
I
1. Add a text element
2. Add a longer word in
3. Resize the width of the element smaller and wider again
4. Verify that the longer word is breaking into multiple lines and not overflowing
5. Enter edit mode, verify that the view is the same for edit mode and display mode -- the word is breaking the same way.


II
1. Do the same steps again but this time use Highlight as the background mode for the Text element.
---



Fixes #1951
